src/content/guides/minimal-startup.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Minimal Startup
+sort: 2
+contributors:
+  - msheliga1
+---
+
+This tutorial uses NPM, webpack and JavaScript to teach basic webpack.
+The webpack command is used to bundle and convert, or transpile, JavaScript modules.
```

Comment:
when writing commands in text, wrap them in \` to achieve: `webpack` command
Also, i can be wrong but it seems last comma isn't needed here?